packages/storybook/stories/Examples/TeamsInterop/snippets/Lobby.snippet.tsx

```diff
@@ -30,15 +29,22 @@ export const Lobby = (props: LobbyProps): JSX.Element => {
     root: { background: theme.palette.white, minHeight: '4.25rem', alignItems: 'center' }
   };
 
-  const videoStreams = useVideoStreams(1);
-  const videoStreamElement = props.isVideoReady ? videoStreams[0] : null;
-
   return (
     <VideoTile
       styles={videoTileStyles}
       isMirrored={true}
       onRenderPlaceholder={() => <></>}
-      renderElement={<StreamMedia videoStreamElement={videoStreamElement} />}
+      renderElement={
+        <StreamMedia
+          videoStreamElement={
+            props.isVideoReady
+              ? ({
+                  // Replace with your video stream element.
```

Comment:
This will cause the lobby component to crash when show video is enabled:
![image](https://user-images.githubusercontent.com/2684369/128568030-dfcd0a50-2d87-42a4-9ee0-ac138fe9aa31.png)

Is there a way to still use the webcam in a minimal sample way?

Reply:
Oops. Missed that

Reply:
Reverted this for now. Added some comments. But I don't think there is a simple solutions to this